manifests/ray-on-eks/core-modules.yaml

```diff
@@ -36,18 +36,18 @@ parameters:
     value:
       eks_nodegroup_config:
         - eks_ng_name: ng1
-          eks_node_quantity: 2
-          eks_node_max_quantity: 5
+          eks_node_quantity: 1
+          eks_node_max_quantity: 1
```

Comment:
Just checking the changes here and in the `ray-cluster-modules.yaml` are intentional?

Reply:
Yep, aligning values a bit between the two manifests